charts/boxplot/src/boxplot-view.js

```diff
@@ -769,10 +770,12 @@ const BoxPlot = ChartView.extend('BoxPlot', {
         brushComponents: ['box-marker'],
       },
 
-      brushActions: this._dependentActions.gestures,
+      brushActions: this._dependentActions?.gestures,
 
       // ref-lines
       refLines: layout.refLine && layout.refLine.refLines,
+      ...getAxisTitleSettings(chartID, theme, layout),
+      ...getAxisLabelSettings(chartID, theme, layout),
```

Comment:
I would like to not use ... here to reduce the risk of future property name collision.
(also apply to the other charts)

Reply:
Fixed.